airbyte-integrations/connectors/source-looker/source_looker/spec.json

```diff
@@ -9,9 +9,15 @@
     "properties": {
       "domain": {
         "type": "string",
-        "pattern": ["^[a-zA-Z0-9._-]*\\.looker\\.com$"],
-        "examples": ["domainname.looker.com"],
-        "description": "Domain for your Looker account, e.g. airbyte.cloud.looker.com"
+        "pattern": [
+          "(^[a-zA-Z0-9._-]*\\.looker\\.com$)|(^(?:[a-zA-Z0-9._-]*)?[a-zA-Z0-9._-]*\\.[a-zA-Z0-9._-]*$)|(^\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}(?::\\d{1,5})?$)"
```

Comment:
can you create unit tests to verify this works? I'm not sure the second regex works as intended. I tested it on https://regex101.com/ with `looker.airbyte.com` and it didn't work.

Reply:
(^(?:[a-zA-Z0-9._-]*)?[a-zA-Z0-9._-]*\\.[a-zA-Z0-9._-]*$) -->pattern in spec.json

(^(?:[a-zA-Z0-9._-]*)?[a-zA-Z0-9._-]*\.[a-zA-Z0-9._-]*$) --->pattern in python(used for testing in regexp101)

Did you try changing \\. to \. and then test it on regexp101. since '.' have special meaning ..we use '\' escape character in python. But regexp pattern in spec.json file expects us to use '\\' as an escape character. Please find the snap below that matches several domains and let me know if any changes required:

![image](https://user-images.githubusercontent.com/35266407/120425205-6d816880-c38b-11eb-9fca-cc53bcf86649.png)

Reply:
@Janardhanpoola  hm, what about `my.shiny.website.com:1234`?

Reply:
agree, please add a unit test for this.